kebechet/services.py

```diff
@@ -46,14 +46,14 @@ def _encode_fslash(slug: str):
             "service_type": ServiceType.GITHUB,
             "config_url": "https://raw.githubusercontent.com/{slug}/{branch}/.thoth.yaml",
             "auth": {"header": "Authorization", "value": "token {token}"},
-            "slug_method": _keep_slug.__func__,
+            "slug_method": _keep_slug.__func__,  # type: ignore
         },
         "gitlab": {
             "token": "GITLAB_KEBECHET_TOKEN",
             "service_type": ServiceType.GITLAB,
             "config_url": "https://gitlab.com/api/v4/projects/{slug}/repository/files/.thoth.yaml/raw?ref={branch}",
             "auth": {"header": "Private-Token", "value": "{token}"},
-            "slug_method": _encode_fslash.__func__,
+            "slug_method": _encode_fslash.__func__,  # type: ignore
```

Comment:
```
Callable[[str], str]
```
For both

Reply:
`"slug_method": _encode_fslash.__func__,  # type: Callable[[str], str]`
Just gives syntax error